app/waf/deploy/docker/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: "3"
+services:
+  waf:
+    image: wafaas-env:latest
+    build:
+      context: .
+    container_name: wafaas-dev-container
+    command: ["bash", "-c", "cd /root/app_dir && npm install && npm start"]
```

Comment:
The docker image should ship node_moudels, right?

Reply:
yes. the 'command' will run every time when 'docker-compose'. 'node_modules' would be shipped on host's directory(in the source code folder).